kalite/topic_tools/content_models.py

```diff
@@ -231,6 +231,7 @@ def get_topic_nodes(parent=None, ids=None, **kwargs):
             Item.path,
             Item.slug,
         ).join(Parent, on=(Item.parent == Parent.pk)).where(selector)
+        return values
```

Comment:
I tried cleaning up this function by having only one return statement: https://github.com/aronasorman/ka-lite/commit/05476767fa14d47c34cfb309842a50f23cf69c79

But tests failed, and then I accidentally pushed the previous PR to 0.16.x. This PR reverts that, and that's why the tests are passing in my branch.